documentcloud/documents/views.py

```diff
@@ -911,3 +914,22 @@ def post_process(self, request, document_pk=None):
         post_process.delay(document_pk, request.data)
 
         return Response("OK", status=status.HTTP_200_OK)
+
+
+class FreestandingEntityViewSet(viewsets.ModelViewSet):
+    serializer_class = EntitySerializer
+    queryset = Entity.objects.none()
+
+    @lru_cache()
+    def get_queryset(self):
+        # pdb.set_trace()
+        # TODO: Should everyone be able to view all entities?
+        return Entity.objects.all()
+
+    def perform_create(self, serializer):
```

Comment:
I think we need to think through how entity creation works in more detail, but I am leaning away from having a POST do a get or create, and more toward multiple API calls.  This can be abstracted in the Python SDK if necessary.

I am going to start a notion page to start a broader discussion about if entities should be all global as they are now, or if they should be per user or per organization and what the pros and cons are, and what analysis types they allow for and what we want to use entities for in the future.